scss/reset/_forms.scss

```diff
@@ -137,6 +137,24 @@ select {
 		outline: none;
 	}
 
+	.inverted & {
+		background-color: $C_fieldBGInverted;
+		color: $C_textPrimaryInverted;
+
+			&:hover {
+				border-color: $C_textHintInverted;
+			}
+
+			&[disabled] {
```

Comment:
nitpick

wonder if this can be flipped around to define `&[disabled]` first, then
```
&:hover {
    border-color: $C_textHintInverted;
 
    &[disabled]...		
}
```
not sure which you think is more legible.